ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -1,104 +1,47 @@
-import React, { useContext, useLayoutEffect, useRef, useState } from 'react';
-import { useSelector } from 'react-redux';
+import React, { useContext, useRef } from 'react';
```

Comment:
Is everything removed in this file really meant to go?  Do we not need `supportsEIP1559` checks anymore?

Reply:
We now get rid of EIP-1559 V1 implementation so this file will be used for only legacy transactions. For EIP-1559 now V2 implementation will be the default.